src/constants.rs

```diff
@@ -63,15 +63,25 @@ macro_rules! dw {
             pub const $name: $struct_name = $struct_name($val);
         )+
 
-        impl fmt::Display for $struct_name {
-            fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
-                match *self {
+        impl $struct_name {
+            pub fn static_string(&self) -> Option<&'static str> {
```

Comment:
You could also add something that returns `Cow<'static, str>`  so that the caller can handle the unknown case identically... but then that case doesn't matter much.